app/views/photos/_actions.html.haml

```diff
@@ -1,7 +1,7 @@
 - if can?(:edit, @photo) && can?(:destroy, @photo)
-  .dropdown.float-right.photo-actions
-    %a#harvest-actions-button.btn.dropdown-toggle{"aria-expanded" => "false", "aria-haspopup" => "true", "data-bs-toggle" => "dropdown", type: "button", href: '#'} Actions
-    .dropdown-menu.dropdown-menu-xs{"aria-labelledby" => "harvest-actions-button"}
```

Comment:
This somehow broke; even before I made it a span and fixed the ids.
Works on prod.
Not on staging